sdk/textanalytics/azure-ai-textanalytics/tests/perfstress_tests/README.md

```diff
@@ -40,5 +40,5 @@ These options are available for all perf tests:
 
 ## Example command
 ```cmd
-(env) ~/azure-ai-textanalytics/tests/perfstress_tests> perfstress DetectLanguagePerfStressTest
+(env) ~/azure-ai-textanalytics/tests/perfstress_tests> perfstress DetectLanguagePerfStressTest --parallel=50 --warmup=1
```

Comment:
I am not sure how quickly you will hit the limit but consider adding `--duration` with the number of seconds needed until you are around the limit.

Reply:
i'll leave it for now, can experiment with it more later one